common/realsense-ui-advanced-mode.h

```diff
@@ -5,6 +5,8 @@
 
 #include <librealsense2/rs_advanced_mode.hpp>
 #include <types.h>
+#include <type_traits>
+#include "../../../common/utilities/string/string-utilities.h"
```

Comment:
The two headers are approximately at the same level.
Why is it referred via `"../../../common`